node/test/hyperbahn/rate-limiter.js

```diff
@@ -270,6 +271,7 @@ function runTests(HyperbahnCluster) {
                 send.bind(null, opts),
                 send.bind(null, opts),
                 waitFor(500),
+                send.bind(null, opts),
```

Comment:
Can you explain why this is testing the new behavior? Is this asserting that we've made a call even after triggering the rate limit?